vendor/github.com/mattn/go-isatty/isatty_bsd.go

```diff
@@ -0,0 +1,18 @@
+// +build darwin freebsd openbsd netbsd dragonfly
+// +build !appengine
+
+package isatty
```

Comment:
__[gometalinter]__
* [dupl] duplicate of isatty_linux.go:4-18

<!-- Comment by [SideCI](https://sideci.com) -->